tst/testmanuals.g

```diff
@@ -33,6 +30,8 @@ WriteExamplesTst := function(directory)
         output := OutputTextFile( Concatenation(directory, "/", chname), false );
         SetPrintFormattingStatus( output, false );
 
+        # PrintTo will *overwrite* the file if it exists, making sure
+        # we don't concatenate the same test into a file multiple times
```

Comment:
This comment is incorrect. It is it the `false` in the call to `OutputTextFile` which takes care of that. Other than that, `PRINT_TO_STREAM` and `APPEND_TO_STREAM` are doing exactly the same thing (see PR #1093). Note that the GAP manual even states that `PrintTo` and `AppendTo` for streams both append -- of course, if you think about it, they have no other choice, since that's how streams work -- there is no way to "delete" the content of a stream.